package.json

```diff
@@ -2,11 +2,17 @@
   "author": "SourceJS",
   "version": "0.1.0",
   "description": "Sourcejs.com project website",
+	"scripts": {
+		"build-local": "grunt default",
+		"build-source": "cd ./node_modules/sourcejs && npm run build",
+    "start": "npm run build-local && npm run build-source && node ./node_modules/sourcejs/app.js"
```

Comment:
before moving sourcejs to NPM install flow, let's wait before 0.6.0 release. Please revert lines 7-8 and 15.